main.cpp

```diff
@@ -732,6 +733,8 @@ static void print_help_msl()
 	                "\t[--msl-device-argument-buffer <descriptor set index>]:\n\t\tUse device address space to hold indirect argument buffers instead of constant.\n"
 	                "\t\tComes up when trying to support argument buffers which are larger than 64 KiB.\n"
 	                "\t[--msl-multiview]:\n\t\tEnable SPV_KHR_multiview emulation.\n"
+	                "\t[--msl-disable-layered-rendering]:\n\t\tDon't set [[render_target_array_index]] in multiview shaders.\n"
+					"\t\tUseful for devices which don't support layered rendering. Only effective when --msl-multiview is enabled.\n"
```

Comment:
Formatting. Need to do it manually since the block is marked format off.

Reply:
Probably call this something else to make it clear this is about multiview. I didn't understand it before I read this description.